Stripe/PublicHeaders/STPAPIResponseDecodable.h

```diff
@@ -11,6 +11,11 @@
 @protocol STPAPIResponseDecodable <NSObject>
 
 /**
+ A string representing the object's type. May be nil.
+ */
++ (nullable NSString *)object;
```

Comment:
Do we have to make it a public method? We already do heterogenous parsing like this in the current STPCustomerDeserializer implementation. I would rather keep it something internal if it's only used in this one spot and not really needed by our users.